### ISSUE TITLE
Support aarch64

### ISSUE THREAD

**Issue by Kekun** [flathub/com.github.micahflee.torbrowser-launcher#53]
I'm sure there is a good reason why only `x86_64` is supported, but it'd be great if `aarch64` was too, as I'd love to use tor Browser on my Linux phone. :)

**Comment by AsciiWolf** [comment]
You're right, it would be great. :-) Sadly the official Tor Browser tar.gz archive (that Tor Browser Launcher uses) is x86_64 only. :-(

**Comment by AsciiWolf** [comment]
https://gitlab.torproject.org/tpo/applications/tor-browser-build/-/issues/40091

**Comment by AsciiWolf** [comment]
https://github.com/micahflee/torbrowser-launcher/issues/662

**Comment by Fallyducape** [comment]
Hi bro
we have talk about business but you have deleted the message and i have found here

**Comment by firefoxlover** [comment]
[Tor Browser supports ARM now](https://blog.torproject.org/new-release-tor-browser-120/) is this issue still relevant?
